feature-libs/product-configurator/rulebased/components/attribute/types/checkbox-list/configurator-attribute-checkbox-list.component.ts

```diff
@@ -136,4 +136,11 @@ export class ConfiguratorAttributeCheckBoxListComponent
       this.onHandleAttributeQuantity(eventObject);
     }
   }
+
+  getAriaLabel(
```

Comment:
There is no need to define this method. 
`getAriaLabelGeneric` could be called directly in HTML tree.
See `createAttributeUiKey` in `configurator-attribute-header.component.html`